backend/src/email/index.js

```
@@ -0,0 +1,37 @@
+const nodemailer = require('nodemailer')
+const pug = require('pug')
+const { htmlToText } = require('html-to-text')
+
+const transport = nodemailer.createTransport({
+  host: process.env.EMAIL_HOST,
+  port: process.env.EMAIL_PORT,
+  auth: {
+    user: process.env.EMAIL_USERNAME,
+    pass: process.env.EMAIL_PASSWORD,
+  },
+})
+
+async function send(payload) {
+  try {
+    await transport.sendMail({...payload, text: htmlToText(payload.html)})
+  } catch (err) {
+    console.log(err)
+    throw new Error('There is a problem sending email. Please try again later.')
+  }
+}
+
+async function sendEmailVerification(email, name, token) {
```

Should throw an error in case parameter `token` or `name` isn't provided

---

👌